docs/content/en/contribute/docs/publish/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Published Doc Structure
+description: Structure of the published documentation
+weight: 450
+---
+
+New writing goes to the `main` branch and can be viewed on the Releases -> development dropdown menu.
+We have staging and production levels for our documentation which are as follows:
+
+* **Latest:** official documentation of the current Keptn release
+  * [link](https://lifecycle.keptn.sh):
+      This is the build of the `page` branch.
+
+* **Development:** documentation being staged for the next Keptn release
+  * [link](https://main.lifecycle.keptn.sh):
+   This is the latest build of the `main` branch.
+
+* **Previous versions:** documentation for earlier releases.
+   These are listed at [link](https://github.com/keptn/lifecycle-toolkit/tree/page/docs/content/en).
+
+* **Contribute:** current version of the "Contribute" guide
+   that is available from a tab on the documentation site.
+
+Let's take a look what happens when your changes are merged in `main` and `page` branch respectively.
+
+## Latest -- Official documentation (Production)
+
+This set of documentation pertains to the latest KLT release and resides within an
+isolated branch known as `page`.
+When a new version of KLT is launched, the contents of the `development`
+branch are rolled into this branch.
+Additionally, it's important to recognize that any
+document changes made using the "Edit this page" feature are seamlessly integrated into this branch.
+
+This uses the `latest` label so that links to a doc page
+remain valid across software and documentation updates.
+
+* build: on each push to `page` with documentation changes
+* build-environment: production
+* config folder: [production](./config/production/)
```

Comment:
this link does not work, is it expected ?

Reply:
Fixed